diffpy/pdfmorph/tests/test_morphsequence.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+
+import pytest
+import filecmp
+from pathlib import Path
+
+from diffpy.pdfmorph.pdfmorphapp import create_option_parser, single_morph, multiple_morphs
+
+thisfile = locals().get('__file__', 'file.py')
+tests_dir = Path(thisfile).parent.resolve()
+testdata_dir = tests_dir.joinpath("testdata")
+testsequence_dir = testdata_dir.joinpath("testsequence")
+testsaving_dir = testdata_dir.joinpath("testsaving")
+test_saving_succinct = testsaving_dir.joinpath("succinct")
+test_saving_verbose = testsaving_dir.joinpath("verbose")
+tssf = testdata_dir.joinpath("testsequence_serialfile.json")
+
+
+class TestMorphSequence:
+    @pytest.fixture
+    def setup_default(self):
+        self.parser = create_option_parser()
+        filenames = ["g_174K.gr", "f_180K.gr", "e_186K.gr", "d_192K.gr", "c_198K.gr", "b_204K.gr", "a_210K.gr"]
+        self.testfiles = []
+        for filename in filenames:
+            self.testfiles.append(testsequence_dir.joinpath(filename))
+        return
+
+    def test_morphsequence(self, setup_default):
+        # Parse arguments sorting by field
+        (opts, pargs) = self.parser.parse_args(["--scale", "1", "--stretch", "0",
+                                                     "-n", "--sort-by", "temperature"])
+
+        # Run multiple single morphs
+        single_results = {}
+        morph_file = self.testfiles[0]
+        for target_file in self.testfiles[1:]:
+            pargs = [morph_file, target_file]
+            # store in same format of dictionary as multiple_morphs
+            single_results.update({target_file.name: single_morph(self.parser, opts, pargs, stdout_flag=False)})
+        pargs = [morph_file, testsequence_dir]
+
+        # Run a morph sequence
+        sequence_results = multiple_morphs(self.parser, opts, pargs, stdout_flag=False)
+
+        # Compare results
+        assert sequence_results == single_results
+
+        # Check using a serial file produces the same result
+        s_file = tssf.resolve().as_posix()
+        (opts, pargs) = self.parser.parse_args(["--scale", "1", "--stretch", "0",
+                                                "-n", "--sort-by", "temperature",
+                                                "--serial-file", s_file])
+        pargs = [morph_file, testsequence_dir]
+        s_sequence_results = multiple_morphs(self.parser, opts, pargs, stdout_flag=False)
+        assert s_sequence_results == sequence_results
+
+    def test_morph_outputs(self, setup_default, tmp_path):
```

Comment:
Check if files are saved properly against files in the new `testsaving` directory